control-plane/pkg/apis/messaging/v1beta1/kafka_channel_lifecycle.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1beta1
+
+import (
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+)
+
+// ConditionAddressable is a reference to the Addressable condition in the base package.
+// KafkaChannel uses the common Condition set defined in the base package.
+const (
+	ConditionAddressable apis.ConditionType = "Addressable"
+)
+
+var conditionSet apis.ConditionSet
```

Comment:
shouldn't this condition set be created via `NewLivingConditionSet` same as [here](https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/apis/messaging/v1beta1/kafka_channel_lifecycle.go#L28)

Reply:
Same as https://github.com/aliok/eventing-kafka-broker/blob/first-stab-at-kafka-ch/control-plane/pkg/apis/messaging/v1beta1/kafka_channel_lifecycle.go